### DISCUSSION TITLE
Solved. Is there way to display the whole list of active keyboard hotkeys (be it default hotkey or customized) ?

### DISCUSSION THREAD

**Discussion by andrew-ysk** [smplayer-dev/smplayer#667] [Q&A] [answered]
I have being wondering how to change audio track for video with several audio tracks to choose from.  
Until today i have accidentally saw that someone said default audio track keyboard shortcut is "k".  

I don't know if there is more good keyboard shortcut that i am missing still, hence i wonder is there way to display the whole list of active  keyboard hotkeys (be it default hotkey or customized) . 

Thx

**Comment by andrew-ysk** [comment, marked as answer]
Please delete this post. I have found out where the keyboard shortcut settings are located.

**Reply by cipricus** [reply]
Why don't you tell everybody where they are? :)

**Reply by tdw419** [reply]
smplayer-portable/shortcuts/default.keys 

This file seems to have the list of shortcuts

**Comment by cipricus** [comment]
> smplayer-portable/shortcuts/default.keys
> 
> This file seems to have the list of shortcuts

So, this last comment of yours is the real answer, and it shouldn't be deleted, given others may have need for it.
